ang/actions/APICall.html

```diff
@@ -55,6 +55,11 @@
 							  helpaction='onInfoPress("Error Handling", "id-handle-api-errors", "CRM\/Sqltasks\/Action\/APICall")'>
 			</ordinary-select2>
 
+            <check-box model="model['store_api_results']"
```

Comment:
This could use a `helpaction` with some basic documentation (i.e. where/how API results are stored if you use this).